aries-site/src/data/structures/components.js

```diff
@@ -1745,6 +1746,20 @@ export const components = [
     sections: [],
     relatedContent: [],
   },
+  {
+    name: 'ToggleGroup',
+    available: true,
+    category: 'Controls',
+    description:
+      'A group of buttons that enable the user alternate between similar visual views such as DataTable, Chart List, Grid and Map to see the same content.',
```

Comment:
maybe something more broad that doesnt have to do with DataTable, Chat etc 
A group of buttons that allow the user toggle on or off.

Reply:
Yea, initially I think there was something else written along these lines. I'm open to having something more broad than listing actual example types.

A group of related buttons that enable the user to to toggle on or off.

Reply:
yeah I like that 
`A set of two-state buttons that can be toggled on or off.` it's a little more broad.